custom_components/tuya_local/devices/littepets.yaml

```diff
@@ -0,0 +1,121 @@
+name: Cat litter box
+products:
+  - id: 000004cha8
+    name: LittePets
+primary_entity:
+  entity: switch
+  name: Power
+  icon: "mdi:switch"
+  dps:
+    - id: 1
+      type: boolean
+      name: switch
+secondary_entities:
+  - entity: sensor
+    name: Work Mode
+    class: enum
+    category: diagnostic
+    dps:
+      - id: 2
+        type: string
+        name: sensor
+        mapping:
+          - dps_val: auto_clean
+            value: Auto
+            icon: "mdi:shimmer"
+          - dps_val: manual_clean
+            value: Manual
+            icon: "mdi:broom"
+          - dps_val: sleep
+            value: Sleep
+            icon: "mdi:sleep"
+          - dps_val: uv
+            value: UV
+            icon: "mdi:sun-wireless"
+          - dps_val: deodorization
+            value: Deodorisation
+            icon: "mdi:spray"
+  - entity: switch
+    name: "Manual Clean"
+    icon: "mdi:broom"
+    dps:
+      - id: 3
+        type: boolean
+        name: switch
+
+  - entity: switch
+    name: "Sleep Mode"
+    icon: "mdi:sleep"
+    dps:
+      - id: 10
+        type: boolean
+        name: switch
+        
+  - entity: switch
+    name: Deodorise
+    icon: "mdi:spray"
+    dps:
+      - id: 17
+        type: boolean
+        name: switch
+        
+  - entity: sensor
+    name: Number of times used
+    category: diagnostic
+    dps:
+      - id: 7
+        type: integer
+        name: sensor
+        unit: times
+        
+  - entity: sensor
+    name: Last visit duration
+    icon: "mdi:paper-roll"
+    category: diagnostic
+    dps:
+      - id: 8
+        type: integer
+        name: sensor
+        unit: seconds
```

Comment:
Unit should be `s`, and the entity should be of `class: duration`

Reply:
Done